wiki/osu!wiki_Contribution_Guide/es.md

```diff
@@ -0,0 +1,106 @@
+# Guía de Contribución de la osu!wiki

```

Comment:
`osu!wiki` -> `osu! wiki`
Can you do that for every instance? Including the name of the folder to `osu!_wiki_Contribution_Guide`. A recent PR was pushed with these minor changes.

Reply:
Fixed! Do I have to create a new PR to change the folder location?

Reply:
No, you can do it on this PR. Hope you know what to do.

Reply:
hmm... I'm a little bit stuck. I tried changing the file path but the exclamation mark in the folder's name (!) gets replaced by a dash (-), so it's not letting me change the directory. Is there another way to change it? I'm sorry, I'm still a newbie at this kind of things :P

Reply:
It seems that you are using the web interface. The web is limited to many things like deleting/creating folders.

Doing it locally will work and is better if you plan on making more PRs. You have to download Github Desktop and an editor. It prefers you to download Atom, but you can use other editors. I use Atom btw.

Reply:
thanks for the advice!